core/core-workflow-steps/pom.xml

```diff
@@ -97,6 +97,22 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- ################################ testing dependencies ################################ -->
```

Comment:
Are these really necessary, I do not see any changes, that require these dependencies?

Reply:
No, will reduce them to the minimum.

Reply:
Did this come back? I am still not getting, why you need additional dependencies in this component, you do not change anything in here? Wondering 🤔